build.assets/Dockerfile

```diff
@@ -209,7 +209,7 @@ RUN curl --proto '=https' --tlsv1.2 -fsSL https://sh.rustup.rs | sh -s -- -y --p
     rustup component add rustfmt clippy && \
     if [ "$BUILDARCH" = "amd64" ]; then rustup target add aarch64-unknown-linux-gnu; fi
 # Install wasm-pack for targeting WebAssembly from Rust.
-RUN cargo install wasm-pack
+RUN cargo install wasm-pack --version 0.11.0
```

Comment:
Instead of defining the version in 4 different places can you define it here https://github.com/gravitational/teleport/blob/master/build.assets/versions.mk next to the rust version and reuse it?

Reply:
18ab60c4c08a7eae1d9509368f62c21e5ac3d936